_templates/component/new/test.ejs.t

```diff
@@ -8,11 +8,8 @@ import { shallowRender } from "../../utils/shallowRender"
 
 describe("<%= name %>", () => {
   it("renders", () => {
-    // Arrange
-    const valA = <%= name %>
-
```

Comment:
I know that the `// Arrange` part of this seems superfluous, but I do like it being included in our template to serve as a helpful pattern. Maybe add this back in?